tests/test_electrolyzer.py

```diff
@@ -187,11 +175,154 @@ def energy_system_options(self):
         )
         #  -----------------------------------------------------------------------------------------
 
+    def test_electrolyzer_constant_efficiency(self):
+        """
+        This test is to check the functioning the example with an offshore wind farm in combination
+        with an electrolyzer and hydrogen storage. The electrolyzer is modelled as the option
+        CONSTANT_EFFICIENCY.
+
+        Checks:
+        - The objective value with the revenue included
+        - Check the bounds on the electrolyzer
+        - Check the setpoint for the windfarm
+        - Check the max production profile of the windfarm
+        - Check the electrolyzer inequality constraints formulation
+
+        """
+        import models.unit_cases_electricity.electrolyzer.src.example as example
+        from models.unit_cases_electricity.electrolyzer.src.example import (
+            MILPProblemConstantEfficiency,
+        )
+
+        base_folder = Path(example.__file__).resolve().parent.parent
+
+        solution = run_esdl_mesido_optimization(
```

Comment:
Think about ensuring that in the test case, the electrolyzer will not be used, e.g. when max power to electrolyzr would be smaller than minimum load (factor 100 I believe).
Change the profile to ensure it and check on it, or change the value of theprofile in the problem definition.

Reply:
Changed the wind profile to 49MW, as the minimum load of the electrolyzer is 50MW

Reply:
Ideally the profile is normally above 50MW, but only for one timestep below 50MW, then you are sure that there is not another reason why the electrolyzer is turned off.

Reply:
Ideally the profile is normally above 50MW, but only for one timestep (not the first) below 50MW, then you are sure that there is not another reason why the electrolyzer is turned off.